renovate.json

```diff
@@ -1,8 +1,7 @@
 {
     "$schema": "https://docs.renovatebot.com/renovate-schema.json",
     "extends": [
-      "config:base",
-      "helpers:pinGitHubActionDigests",
+      "github>platform-engineering-org/.github",
```

Comment:
Some thoughts :)

I see inside the `default.json` there is an extension to `schedule:weekly` which is different than our current schedule (we would like to have updates once per sprint). Will this extension affect our current schedule?

Have we tested the new configuration elsewhere to see which will be the behavior of the renovate bot (e.g a forked repo)?

Reply:
It is used by all the other repos supported by the PE team. For example:
github.com/platform-engineering-org
github.com/arcalot

Reply:
One can override the global settings by adding a setting in the local renovate.json file

Reply:
I've created this [PR](https://github.com/platform-engineering-org/renovate-runner/pull/95) in order to be able to check how the runner will behave with the global config before merging. @lmilbaum

Reply:
@lmilbaum thank you for merging the PR I have re-invited the platform-engineering bot in my fork and I'm waiting for the acceptance of the invitation :)

Reply:
@thepetk Done. For some reason GitHub is not sending invite notification emails. Pocking me like you have done is the only way for me to know that such an invite happened. Apologies of the inconvenience.

Reply:
@lmilbaum I've seen there is a change to the current behavior of the bot.

More detailed due to the packageRule defined [here](https://github.com/platform-engineering-org/.github/blob/main/default.json#L14C4-L20C7) we will now include all devfile image updates inside a single PR (example PR opened here: https://github.com/thepetk/registry/pull/76).

For the registry case, as each stack might be owned by different teams, every devfile image update needs to be inside a different PR. Example PRs:
* https://github.com/devfile/registry/pull/241
* https://github.com/devfile/registry/pull/242

This will allow us to communicate each time with the team responsible for the stack and after their review to merge the PR. If we inherit this package rule we will have to wait for all the teams to review a PR in order to be able to merge it.

I've tried to find a workaround in order to skip this package rule but I wasn't so lucky. Any ideas how we can avoid this rule?

Reply:
@thepetk Apologies for the delay. Too many things on my plate. I think the following should work:
```
  "packageRules": [
    {
      "description": "Disable grouping patch and digest updates",
      "matchUpdateTypes": ["patch", "digest"],
      "groupName": null
    }
]

Reply:
I'm resolving the issue and proposing the final changes so the PR can be merged!